doc/ceph-volume/lvm/newdb.rst

```diff
@@ -9,3 +9,48 @@ Logical volume name format is vg/lv. Fails if OSD has already got attached DB.
 Attach vgname/lvname as a DB volume to OSD 1::
 
     ceph-volume lvm new-db --osd-id 1 --osd-fsid 55BD4219-16A7-4037-BC20-0F158EFCC83D --target vgname/new_db
+
+Fixing BlueFS Spillover to Slow Devices
+---------------------------------------
+
+Under certain conditions after upgrades, OSD databases spill onto slow storage
+and the Ceph cluster generates BlueFS spillover warnings. ``ceph health
+detail`` returns these spillover warnings.  Here is an example of a spillover
+warning::
+
+   osd.76 spilled over 128 KiB metadata from 'db' device (56 GiB used of 60 GiB) to slow device
+
+To move these bytes from the slow device to the relatively faster device, take
```

Comment:
These bytes / this DB metadata

Reply:
S/relatively/:

faster is already comparative

Reply:
You're right. You're looking at a Frankensentence here. An earlier draft had two "relatively"s in it.

Reply:
I've used "slower" and "faster" so that it's comparative in both directions.